web_external/app.js

```diff
@@ -1,49 +0,0 @@
-import Backbone from 'backbone';
```

Comment:
Did you remove the web_external on purpose ?

Reply:
Yes I did, it was not longer useful, we will start using new tools as Vue or React.